scripts/build

```diff
@@ -29,7 +29,6 @@ with_pause=${4:-false}
 
 PAUSE_CONTAINER_IMAGE="amazon/amazon-ecs-pause"
 PAUSE_CONTAINER_TAG="0.1.0"
-PAUSE_CONTAINER_TARBALL="amazon-ecs-pause.tar"
```

Comment:
Question; I think I missed something, has this line been removed intentionally?

Reply:
This particular variable is not being used in this script, so I in my opinion it is safe to remove.